### PR TITLE
change styleName to className in Accordion component

### PR DESCRIPTION
**Pull request by sanmiawi**

### CODE REVIEW

**Review by Sakai-san — APPROVED** [review]

File: pyrene/src/components/Accordion/Accordion.tsx [hunk 1]
```diff
@@ -22,7 +23,7 @@ export interface AccordionProps {
 const Accordion: React.FC<AccordionProps> = ({
   sections = [],
 }: AccordionProps) => (
-  <div styleName={`accordion ${isCustomAccordion(sections) ? 'accordion-custom' : 'accordion-regular'}`}>
+  <div className={clsx(styles.accordion, { [styles.accordion_custom]: isCustomAccordion(sections), [styles.accordion_regular]: !isCustomAccordion(sections) })}>
```
Comment (justmejulian) [inline pyrene/src/components/Accordion/Accordion.tsx:26]: Why the change?
This way isCustomAccordion is run twice
